packages/africa/africa.svg

```diff
@@ -0,0 +1,245 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Could you clean the SVG file to follow [the guidelines](https://github.com/VictorCazanave/svg-maps/blob/master/CONTRIBUTING.md#svg-file) please?